DEFAULT_CONFIG.json5

```
@@ -138,6 +138,17 @@
       /// or different values for router, peer or client mode (e.g. autoconnect: { router: [], peer: ["router", "peer"] }).
       /// Each value is a list of: "peer", "router" and/or "client".
       autoconnect: { router: [], peer: ["router", "peer"], client: ["router"] },
+      /// Strategy for autoconnection, mainly to avoid nodes connecting to each other redundantly.
+      /// Possible options are:
+      /// - "always": always attempt to autoconnect, may result in redundant connection which will then be closed.
+      /// - "greater-zid": attempt to connect to another node only if its own zid is greater than the other's.
+      ///   If both nodes use this strategy, only one will attempt the connection.
+      ///   This strategy may not be suited if one of the nodes is not reachable by the other one, for example
+      ///   because of a private IP.
+      /// Accepts a single value (e.g. autoconnect: "always") which applies whatever node would be auto-connected to,
+      /// or different values for router, peer, or client depending on the type of node detected
+      /// (e.g. autoconnect_strategy : { router:  "always", peer: "greater-zid", client: "always" })
+      autoconnect_strategy: { router:  "always", peer: "always", client: "always" },
```

Client cannot be an autoconnect target. Please update the comments accordingly, and add a check on the config to disallow clients here.
```suggestion
      /// Accepts a single value (e.g. autoconnect: "always") which applies whatever node would be auto-connected to,
      /// or different values for router and/or peer depending on the type of node detected
      /// (e.g. autoconnect_strategy : { router:  "always", peer: "greater-zid" })
      autoconnect_strategy: { router:  "always", peer: "always" },
```